test/test.js

```diff
@@ -3196,14 +3186,12 @@ describe('array checks', function() {
             var arr2 = [10, 6, 7, 8, 9];
             expect(is.any.sorted(arr1, arr2)).to.be.true;
             expect(is.any.sorted([arr1, arr2])).to.be.true;
-            // extended behaviour not available with 'any' interface
```

Comment:
Ok, so which one is best option.
* Keep that comment, but at the top
* Remove that comment
* Something else :stuck_out_tongue: